src/test/resources/application.yml

```diff
@@ -104,6 +104,9 @@ akhq:
           - connect/update
           - connect/delete
           - connect/state/update
+        attributes:
```

Comment:
Sorry, this might be a stupid question. If we remove this, `void findAllByUser()` in `AccessControlRepositoryTest.java` will fail, right...? (` assertEquals(5, searchResult.getAcls().size());` would become ` assertEquals(7, searchResult.getAcls().size());`)